build_domain.py

```diff
@@ -1,31 +1,46 @@
 """ This py build summary md from yaml files """
 
 import os
+import logging
 
 import pandas as pd
 import yaml
+logging.basicConfig(filename='example.log', encoding='utf-8', level=logging.DEBUG)
 
 def count_yaml_keys(file_path):
     """count"""
-    with open(file_path, "r", encoding="utf-8") as yaml_file:
-        data = yaml.safe_load(yaml_file)
-        if data is None:
-            return 0
-        return len(data.keys())
+    logging.info('running count_yaml_keys ')
+    try:
+        with open(file_path, "r", encoding="utf-8") as yaml_file:
+            data = yaml.safe_load(yaml_file)
+            if data is None:
+                logging.warning('file is empty')
+                return 0
+            return len(data.keys())
+    except:
+        logging.fatal('Something wrong when open the file path. Please check')
 
 
 def get_yaml_content(file_path):
-    with open(file_path, "r", encoding="utf-8") as yaml_file:
-        return yaml.safe_load(yaml_file)
+    logging.info('running get_yaml_content ')
+    try:
+        with open(file_path, "r", encoding="utf-8") as yaml_file:
+            return yaml.safe_load(yaml_file)
+    except IOError as e:
+        logging.fatal(e)

```

Comment:
When logging fatal, please check if you can handle the fatal error. With this piece of codes, you did nothing after you raised the error. Is it okay?